hooks/aci_opflex_utils.py

```diff
@@ -213,7 +213,7 @@ def create_opflex_interface():
 auto %s.%s
 iface %s.%s inet dhcp
 vlan-raw-device %s
-post-up /sbin/route -nv add -net 224.0.0.0/4 dev %s.%s
+post-up ip route add  224.0.0.0/240.0.0.0 dev %s.%s
```

Comment:
Shouldn't this also be /sbin/ip ?

(py36) vagrant@ubuntu-bionic:~/work/openstack/upstream/group-based-policy$ which ip
/sbin/ip

Reply:
Sounds good - thanks for the explanation!